Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean

```diff
@@ -86,240 +86,329 @@ local macro "pderiv_simp" : tactic =>
     pderiv_X_of_ne (by decide : (2 : Fin 3) ≠ 0), pderiv_X_of_ne (by decide : (0 : Fin 3) ≠ 2),
     pderiv_X_of_ne (by decide : (2 : Fin 3) ≠ 1), pderiv_X_of_ne (by decide : (1 : Fin 3) ≠ 2)])
 
-variable {R : Type u} [CommRing R] (W : Jacobian R)
+variable {R : Type u} [CommRing R] {V : Jacobian R} {F : Type u} [Field F] {W : Jacobian F}
```

Comment:
Why not `F : Type v`? Are there definitely no results which use both R and F here?

Reply:
Oops, thanks!